sections/semantics-scriptings.include

```diff
@@ -1421,7 +1418,7 @@ o............A....e
 
   When authors use the <{canvas}> element, they must also provide content that, when
   presented to the user, conveys essentially the same function or purpose as the
-  <code>canvas</code>' bitmap. This content may be placed as content of the <code>canvas</code>
+  <code>canvas</code>'s bitmap. This content may be placed as content of the <code>canvas</code>
```

Comment:
Not critical, but worth keeping in mind for long-term spec hygiene: Whenever possible, let's try to eliminate `<code></code>` markup (unless it's specifically denoting some code). These are hold-overs from the old WHATWG source, where `<code>` is used for linking (in Anolis). With Bikeshed, we use a variety of syntaxes for linking. Here, `<code>canvas</code>` is a reference to the element, so we can use the element syntax: `<{canvas}>` instead. It's good to improve linking wherever we can as we make changes :-)